Taggers/interface/CollectionDumper.h

```diff
@@ -367,14 +377,13 @@ namespace flashgg {
                 }
             }
         }
-        if (splitPdfByStage0Cat_ && dumpPdfWeights_) {
+        if (reweighGGHforNNLOPS_) {
```

Comment:
@threiten this might come in handy for differentials as well.

Reply:
Is this actually doing the reweighting (i.e. multiplying the NNLOPS weights with weights) or just dumping the NNLOPS weights ?

Reply:
It actually applies the reweighting; this is already present in the legacy branch however, see [here](https://github.com/cms-analysis/flashgg/blob/dev_legacy_runII/Taggers/interface/CollectionDumper.h#L615-L621)

Reply:
Ok thanks. The version of the CollectionDumper is a little different there in the differentials branch [(see here)](https://github.com/threiten/flashgg/blob/a95a194d48fbeb271f7d49d3e8c6889d359459df/Taggers/interface/CollectionDumper.h#L638-L653), where it is in fact just dumped. In the version that is in this PR now, the NNLOPSweight will only be applied of splitPdfByStage0Bin_ || splitPdfByStage1Bin_ and reweighGGHforNNLOPS_ are true, which is not true for us. So we might want to find a solution that will apply the NNLOPSweight also if only reweighGGHforNNLOPS_ is true.